src/xhr.ts

```diff
@@ -0,0 +1,47 @@
+import { IPersonData } from './template';
+
+const CREATED_RESPONSE_STATUS = 201;
+
+export function createSession<ICreationSession>(
+  host: string,
+  apiKey: string,
+  data: { person?: IPersonData; vendorData?: string },
+  cb: (statusObject, resp) => void
+): void {
+  const url = `${host}/v1/sessions`;
+  const xhr = new XMLHttpRequest();
+  xhr.open('POST', url, true);
+  xhr.setRequestHeader('Content-type', 'application/json');
+  xhr.setRequestHeader('x-auth-client', apiKey);
```

Comment:
one more thing. let's set 'x-origin': 'js-sdk' to better analyze session sources